lib/asyncapiSchemaFormatParser.js

```diff
@@ -81,17 +81,14 @@ function getValidator(version) {
  * 
  * @private
  * @param {Object} asyncapiSchema AsyncAPI specification JSON Schema
- * @param {Object} version AsyncAPI version.
  * @returns {Object} valid JSON Schema document describing format of AsyncAPI-valid schema for message payload
  */
-function preparePayloadSchema(asyncapiSchema, version) {
-  const payloadSchema = `http://asyncapi.com/definitions/${version}/schema.json`;
+function preparePayloadSchema(asyncapiSchema) {
   const definitions = asyncapiSchema.definitions;
-  // Remove the meta schemas because it is already present within Ajv, and it's not possible to add duplicate schemas.
-  delete definitions['http://json-schema.org/draft-07/schema'];
-  delete definitions['http://json-schema.org/draft-04/schema'];
```

Comment:
no need to remove it as we do not use `http://json-schema.org/draft-07/schema` in $id any longer